drivers/docker/driver_test.go

```diff
@@ -1140,6 +1145,9 @@ func TestDockerDriver_MACAddress(t *testing.T) {
 		t.Parallel()
 	}
 	testutil.DockerCompatible(t)
+	if runtime.GOOS == "windows" {
+		t.Skip("Windows docker does not support setting MacAddress")
```

Comment:
This is currently failing unexpectedly for me.  The tests fails with seeing an empty mac string from API:

```
    driver_test.go:1155: 
        	Error Trace:	driver_test.go:1155
        	Error:      	Not equal: 
        	            	expected: "00:16:3e:00:00:00"
        	            	actual  : ""
        	Test:       	TestDockerDriver_MACAddress
```